### PR TITLE
PR to resolve Feature card - 009 [Automation] issue #224

### PR DESCRIPTION
**Pull request by vishal-kumar3**
This PR resolves the issue #224

https://github.com/user-attachments/assets/d40fcb24-0046-468d-817d-2a34db57f3bf

### CODE REVIEW

File: content/docs/progress/animatedtimeline.mdx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+title: Animated Timeline
+description: The Animated Timeline component is an interactive, visually appealing timeline that responds to user interaction. Built with Framer Motion and React, this component highlights key events or milestones in a vertical timeline structure.When a user hovers over a specific timeline item, the associated circular dot and all previous dots in the sequence turn green, indicating progression. The dot size also enlarges slightly, enhancing the focus on the current event. The component offers a sleek and smooth animation experience, perfect for showcasing chronological steps, milestones, or achievements in an engaging and user-friendly manner.This component is highly customizable, allowing easy modifications to the timeline content and styling, making it suitable for diverse applications such as resumes, project timelines, or product roadmaps.
+author: YourTwitterUsername
```
Comment (hari) [inline content/docs/progress/animatedtimeline.mdx:4]: @vishal-kumar3 Update this with your username.

Comment (vishal-kumar3) [reply content/docs/progress/animatedtimeline.mdx]: Done!

**Comment by sudhashrestha** [comment]
Hi @vishal-kumar3 Please link the issue to this PR.

**Comment by vishal-kumar3** [comment]
> Hi @vishal-kumar3 Please link the issue to this PR.

Changed the title and description mentioning to issue #224. Is it correct?

**Comment by hari** [comment]
@vishal-kumar3 Can you make the padding better? Increase vertical padding and decrease horizontal padding to make it symmetrical.

Also, the circle and the title are not aligned, which seems correct as per the referenced design, but I think we should align them with ours.

**Comment by vishal-kumar3** [comment]
> @vishal-kumar3 Can you make the padding better? Increase vertical padding and decrease horizontal padding to make it symmetrical.
> 
> Also, the circle and the title are not aligned, which seems correct as per the referenced design, but I think we should align them with ours.

fixed the alignment issue and the padding.

**Review by hari — APPROVED** [review]

**Pull request merged**